torch/_inductor/codegen/wrapper.py

```diff
@@ -984,6 +1000,12 @@ def write_wrapper_decl(self):
                 torch::List<c10::optional<at::Scalar>> optional_list;
                 """
             )
+            if not V.graph.aot_mode:
+                self.wrapper_call.splice(
+                    """
+                    cudaStream_t stream;
```

Comment:
This should go to CudaWrapperCodeGen. CppWrapperCodeGen is shared for cpu and gpu. I know we are already mixing the logic at some places, but let's not add more to that.

Reply:
@desertfire , sure. I put it here to keep consistency because we write the stream param in the parameter list at the top of this function (line 950). So by the same logic, shouldn't that go in CudaWrapperCodeGen as well? If so, do you want me to refactor this write_wrapper_decl method and specialize in CudaWrapperCodeGen vs. CppWrapperCodeGen?